index.html

```diff
@@ -1,52 +1,3336 @@
+                data-name="Mobile Portfolio"
+                class="q17"
+              ><div>Mobile Portfolio</div>
+              </div>
+              </div>
+              <div class="outer_q18">
+              <div
+                id="q18"
+                data-name="Group 24"
+                class="q18"
+              >
+                <div class="outer_q19">
+                <div
+                  id="q19"
+                  data-name="Rectangle 27"
+                  class="q19"
+                >
+                </div>
+                </div>
+                <div class="outer_q20">
+                <div
+                  id="q20"
+                  data-name="JavaScript"
+                  class="q20"
+                ><div>JavaScript</div>
+                </div>
+                </div>
+              </div>
+              </div>
+              <div class="outer_q21">
+              <div
+                id="q21"
+                data-name="Group 23"
+                class="q21"
+              >
+                <div class="outer_q22">
+                <div
+                  id="q22"
+                  data-name="Rectangle 27"
+                  class="q22"
+                >
+                </div>
+                </div>
+                <div class="outer_q23">
+                <div
+                  id="q23"
+                  data-name="HTML/CSS"
+                  class="q23"
+                ><div>HTML/CSS</div>
+                </div>
+                </div>
+                <div class="outer_q24">
+                <div
+                  id="q24"
+                  data-name="Ruby on Rails"
+                  class="q24"
+                ><div>Ruby on Rails</div>
+                </div>
+                </div>
+              </div>
+              </div>
+              <div class="outer_q25">
+              <div
+                id="q25"
+                data-name="Rectangle 27"
+                class="q25"
+              >
+              </div>
+              </div>
+              <div class="outer_q26">
+              <div
+                id="q26"
+                data-name="Group 19"
+                class="q26"
+              >
+                <div class="outer_q27">
+                <div
+                  id="q27"
+                  data-name="Rectangle 20"
+                  class="q27"
+                >
+                </div>
+                </div>
+                <div class="outer_q28">
+                <img
+                  id="q28"
+                  data-name="ic_arrow_right"
+                  src="images/ic_arrow_right_5.png"
+                  alt="ic_arrow_right"
+                  class="q28"
+                >
+                </div>
+                <div class="outer_q29">
+                <div
+                  id="q29"
+                  data-name="See this project"
+                  class="q29"
+                ><div>See this project</div>
+                </div>
+                </div>
```

Comment:
- [ ] Kindly change all occurrences of `See this Project` to a `button` or `link`. Also, set some visual interactions to the element. The previous reviewer mentioned the three states to implement. Kindly work on it.